modules/nlu-testing/src/module_lifecycle/on_bot_mount.ts

```diff
@@ -0,0 +1,17 @@
+import * as sdk from 'botpress/sdk'
+
+import { BotState, VisuState } from '../backend/typings'
+
+export function getOnBotMount(state: VisuState) {
+  return async (bp: typeof sdk, botId: string) => {
+    state[botId] = {} as BotState
+    state[botId].ghost = bp.ghost.forBot(botId)
+
+    const axiosConfig = await bp.http.getAxiosConfigForBot(botId)
+    state[botId].axiosConfig = axiosConfig
+    state[botId].botId = botId
+    state[botId].language = (await bp.bots.getBotById(botId)).defaultLanguage
+    state[botId].trainDatas = []
+    state[botId].testDatas = []
+  }
```

Comment:
What do you think of this format ? 

```js
state[botId] = {
  ghost: bp.ghost.forBot(botId),
  axiosConfig: await bp.http.getAxiosConfigForBot(botId),
  language = (await bp.bots.getBotById(botId)).defaultLanguage,
  trainDatas: [],
  testDatas: [],
  botId // shouldn't be necessary, since it's the key of your object
}
```

Also, you are missing localUrl